driver-core/pom.xml

```diff
@@ -85,6 +85,13 @@
     </dependency>
 
     <dependency>
+      <groupId>com.github.jnr</groupId>
+      <artifactId>jnr-ffi</artifactId>
+      <version>2.0.7</version>
```

Comment:
We might want to extract the version as a property, for the reasons explained in 0853ed8.